<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortWindow</class>
 <widget class="QDialog" name="SortWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="top_tags">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>40</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 12pt 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>当前堆顶元素</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>37</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="top_number">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>10</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
  </widget>
  <widget class="QLabel" name="heap_cur">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>178</y>
     <width>501</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="heap_title">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>140</y>
     <width>108</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>当前堆内情况</string>
   </property>
  </widget>
  <widget class="QLabel" name="array_cur">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>501</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="array_title">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>350</y>
     <width>126</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>当前数组内情况</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
